terraform-random-pet/README.md

```diff
@@ -0,0 +1,102 @@
+# terraform-random-pet
```

Comment:
I wonder if we should rename the directory to `child-module` so that it follows what we're doing with the root module? I'm not sure about this though -- what do you think?

Reply:
I suggest showing how to name a child module correctly according to its content. So we kinda follow our own recommendations. I don't feel strongly about that, as it may be confusing. I hoped that README would elaborate on why we use `random` here. Does it make sense to you?

Reply:
Agreed 👍